src/OpenTelemetry.Api/README.md

```diff
@@ -434,6 +434,75 @@ and
 [extract](../../examples/MicroserviceExample/Utils/Messaging/MessageReceiver.cs)
 context.
 
+## Introduction to OpenTelemetry .NET Metrics API
+
+Metrics in OpenTelemetry .NET is a somewhat unique implementation of the
+OpenTelemetry project, as the Metrics API are incorporated directly into the
+.NET runtime itself, as part of the
+[`System.Diagnostics.DiagnosticSource`](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/6.0.0-rc.1.21451.13)
+package. This means, users can instrument their applications/libraries to emit
+metrics by simply using the `System.Diagnostics.DiagnosticSource` package. This
+package can be used in applications targetting any of the officially supported
+versions of [.NET Core](https://dotnet.microsoft.com/download/dotnet-core), and
+[.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) except
+for versions lower than `.NET Framework 4.6.1`.
+
+## Instrumenting a library/application with .NET Metrics API
+
+### Basic metric usage
+
+1. Install the `System.Diagnostics.DiagnosticSource` package version
```

Comment:
Is this going to be the only way to instrument applications, or is it still possible to include the `OpenTelemetry.Api` package?

Reply:
the only way. There are no plans to add shim/wrapper around this in the API package.

Reply:
So that means that if I want to add metrics to my library, I need to take a dep on the DiagnosticSource package and not OTel? I can see why that, but I feel it would be also good to have a wrapper, not only because it goes against the spec, but other people coming from other languages have to learn, yet this new thing which is not so obvious. 🤔

Reply:
If you take a dependency on Otel.API, then it automatically brings in DS package.
This is the same model we use for tracing (and logging) as well.
We added wrapper for tracing, because the Activity API had lot of naming differences (because Activity existed before OTel came into existence.), and many folks felt the need for a wrapper. 

Metric API in .NET was built ground up, closely following OTel spec, so there are not many differences, warranting the cost of maintaining another wrapper. Having said that, if there is any strong reason to have a wrapper - please create an issue and we can discuss more.